app/platform_stats/rest.py

```diff
@@ -102,12 +106,32 @@ def get_usage_for_all_services():
     for service_id, breakdown in lb_by_service:
         combined[service_id]['letter_breakdown'] += (breakdown + '\n')
 
+    billing_details = fetch_billing_details_for_all_services()
+    for service in billing_details:
+        if service.service_id in combined:
```

Comment:
Can this be simplified by removing the if statement and just use:
```
combined[service.service_id].update({
   'purchase_order_number': service.service_purchase_order_number or service.organisation_purchase_order_number,
...
})```

Reply:
Not sure, what would happen if service_id is not in combined? Since not all services have bills to pay? 
Wouldn't that break?

Reply:
Ahh yes, that would be nicer, but then I think it would not be exactly as requested in the ticket:

> In order to determine where to fetch these values from, first check if there is a ‘purchase order number’ present for the service. If there is then set these fields using the values stored against the service.

> If there is no purchase order number on the service, then check if there is a purchase order number on the parent organisation. If there is then set these fields using the values stored against the organisation.

> If neither is present leave those fields empty in the report